crates/tests/src/e2e/ibc_tests.rs

```diff
@@ -579,10 +579,13 @@ fn pgf_over_ibc_with_hermes() -> Result<()> {
 
 #[test]
 fn proposal_ibc_token_inflation() -> Result<()> {
+    const MASP_EPOCH_MULTIPLIER: u64 = 2;
```

Comment:
This param seems to make the test fail (if we set it to 1 instead it works). @yito88 investigated the issue and found out that an update to Hermes solves it and makes this test pass